test/src/test/java/org/springframework/security/test/web/configurers/UrlAuthorizationConfigurerAnonymousTest.java

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
The checkstyle fails. You can run it using `./gradlew checkstyleMain checkstyleTest`.

Reply:
You're right. The issue was that the file was indented by white spaces rather than tabs.